SmartCarMQTTApp/app/src/main/java/com/example/smartcarmqttapp/screens/quiz/QuizQuestionActivity.java

```diff
@@ -356,9 +359,10 @@ public void addQuestion(List<Question> questionList){
         currentQuestionNum++;
         questionCountText.setText(currentQuestionNum + " / " + quizState.getQuestions().size());
         scoreText.setText(Integer.toString(scoreNumber));
+        questionText.setText(currentQuestion.getQuestion());
         categoryText.setText(currentQuestion.getCategory());
-
-        //TODO Set illustration to imageview!!!!!!!!!!!!!!
+        
+        questionImage.setBackgroundResource(currentQuestion.getImage());
```

Comment:
What if there is no image for the question? Wouldn't this introduce a bug since you are setting the imageView to a NULL background resource

Reply:
There is always an image for a question. A question must have an image as it is one of the parameters.